tools/code_format/config.yaml

```diff
@@ -421,3 +421,4 @@ visibility_excludes:
 - source/extensions/config_subscription/grpc/BUILD
 - source/extensions/load_balancing_policies/subset/BUILD
 - source/extensions/load_balancing_policies/ring_hash/BUILD
+- source/extensions/resource_monitors/downstream_connections/BUILD
```

Comment:
Is this change necessary? I think this is one of the files we are keeping "in sync" with Envoy and they don't seem to be including `downstream_connections` here.